src/arrays.js

```diff
@@ -1,61 +1,72 @@
 const getNthElement = (index, array) => {
-  // your code here
+if (index > array.length -1) {
+  return array[index % array.length]
+} else { return array[index]}
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  return array.toString()
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',')
 };
 
 const addToArray = (element, array) => {
-  // your code here
+   array.push(element)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return newArray2 = array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.map(strings => strings.toString());
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+  return strings.map(uppercase => uppercase.toUpperCase());
 };
 
 const reverseWordsInArray = (strings) => {
-  // your code here
+    return strings.map(reversed => reversed.split("").reverse().join(""));
 };
 
 const onlyEven = (numbers) => {
-  // your code here
+    return numbers.filter(number => number % 2 === 0 )
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  return array.filter(arrFil => arrFil !== array[index]);
 };
 
 const elementsStartingWithAVowel = (strings) => {
-  // your code here
-};
+    return strings.filter(arrFil => arrFil.charAt(0) === 'a' ||
+    arrFil.charAt(0) === 'e' || arrFil.charAt(0) === 'i' || 
+    arrFil.charAt(0) === 'o' || arrFil.charAt(0) === 'u' || 
+    arrFil.charAt(0) === 'A' || arrFil.charAt(0) === 'E' || 
+    arrFil.charAt(0) === 'I' || arrFil.charAt(0) === 'O' ||
+    arrFil.charAt(0) === 'U')
+  };
 
 const removeSpaces = (string) => {
-  // your code here
+    return string.replace(/\s/g,'');
+  
 };
 
 const sumNumbers = (numbers) => {
-  // your code here
+      return numbers.reduce(function(total,curVal) {
+       return  total + curVal;
+     })
 };
 
 const sortByLastLetter = (strings) => {
-  // your code here
+    return strings.sort((cur,next) => cur.charCodeAt(cur.length-1) -
```

Comment:
If your lines start to get too long (aim for 80 characters, 120 at most), then consider not using an implicit return within an arrow function:

```js
return strings.sort((current, next) => {
  return current.charCodeAt(current.length - 1) - next.charCodeAt(next.length - 1));
});
```

Reply:
This is still 85 so variable assignment might be a better option still:

```js
return strings.sort((current, next) => {
  const currentLastLetter = current.charCodeAt(current.length - 1);
  const nextLastLetter = next.charCodeAt(next.length - 1));

  return currentLastLetter - nextLastLetter;
});
```

Never sacrifice readability for shorter lines (note use of `current` instead of `cur`).